code/modules/surgery/organs/organ.dm

```diff
@@ -116,9 +110,6 @@ var/list/organ_cache = list()
 		if(germ_level >= INFECTION_LEVEL_THREE)
 			necrotize()
 
-		if(damage >= max_damage)
```

Comment:
this is done here, then again below; this is redundant to have it necrotize twice.